mesh/mesh.cpp

```diff
@@ -5230,17 +5230,24 @@ void Mesh::LoadPatchTopo(std::istream &input, Array<int> &edge_to_knot)
 
    input >> ident; // 'edges'
    input >> NumOfEdges;
-   edge_vertex = new Table(NumOfEdges, 2);
-   edge_to_knot.SetSize(NumOfEdges);
-   for (int j = 0; j < NumOfEdges; j++)
+   if (NumOfEdges > 0)
```

Comment:
To help me understand the new code here, is there an example that has `NumOfEdges == 0`?

Reply:
Hi @dylan-copeland, I have added a test and sample run. 
```
nurbs_ex1 -m ../../data/two-squares-nurbs-autoedge.mesh -o 1 -rf ../../data/two-squares.ref
```
For usage you basically have leave the section `edges` in the mesh file empty.